src/wallet/storage.rs

```diff
@@ -48,11 +51,14 @@ impl WalletStore {
         network: Network,
         seedphrase: String,
         passphrase: String,
+        wallet_birthday: Option<u64>,
     ) -> Result<Self, WalletError> {
         let mnemonic = Mnemonic::parse(seedphrase)?;
         let seed = mnemonic.to_seed(passphrase);
         let master_key = ExtendedPrivKey::new_master(network, &seed)?;
 
+        log::error!("Wallet is here: {:?}", path);
```

Comment:
why this error?

Reply:
Random stuff which I added for getting the value. Removed in next commit